app/views/users/show.html.erb

```diff
@@ -1,78 +1,65 @@
   <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet">
 </head>
 <body>
-<div class="container ">
-  <div class="row">
-    <div class="col-sm-12  ">
-      <h1 class="mt-2">User Details</h1>
-      <p class="lead">Here you can see all the users of the blog.</p>
-    </div>
-  </div>
+<div class="container py-4">
+
   <div class="row">
-    <div class="col-sm-2 border rounded">
-      <div class="user-photo  mt-5">
-        <% if @user.photo && @user.photo.match(/\.(png|jpg|jpeg|gif)$/i) %>
-          <img src="<%= @user.photo %>" alt="User Photo" style="width: 150px">
-        <% else %>
-          User<%= @user.id %> Pic
-        <% end %>
+    <div class="col-12">
+      <div class="user-info d-flex justify-content-between">
+        <div class="mr-3">
+          <% if @user.photo && @user.photo.match(/\.(png|jpg|jpeg|gif)$/i) %>
+            <img src="<%= @user.photo %>" alt="User Photo" class="user-photo" style="width: 80px; height: 80px;">
+          <% else %>
+            <div class="user-photo" style="width: 150px; height: 150px;">User<%= @user.id %> Pic</div>
+          <% end %>
+        </div>
+        <div class="d-flex justify-content-between border p-3 w-75">
+          <h1><%= @user.name %></h1>
+          <p class="mt-4"><%= link_to "Number of posts: #{@user.posts_counter}", user_posts_path(@user), class: "text-muted" %></p>
+        </div>
       </div>
     </div>
-    <div class="col-sm-10 border rounded">
-      <h5 class="user-name mt-5">
-        <%= @user.name %>
-      </h5>
-      <p class="user-posts text-right">
-        <%= link_to "Number of posts: #{@user.posts_counter}", user_posts_path(@user), title:"To User Posts" %>       </p>
-    </div>
   </div>
-</div>
 
-<div class="container ">
+  <!-- Bio Section -->
   <div class="row">
-    <div class="col-sm-12 border rounded my-3">
-      <h5 class="user-name py-3">
-        <%= @user.name %>  Bio
-      </h5>
-      <p class="user-name ">
-        <%= @user.bio %>
-      </p>
+    <div class="col-12 my-3">
+      <div class="p-3 border rounded bio-section">
+        <h5><%= @user.name %> Bio</h5>
+        <p><%= @user.bio %></p>
+      </div>
     </div>
   </div>
-</div>
 
-<div class="container">
-  <div class="row">
-    <% @posts.each do |post| %>
-      <div class="col-sm-12 border rounded my-3">
-        <h5 class="user-name py-3">
-          Post<%= post.id %>
-        </h5>
-        <h3 class="user-name">
-          <%= post.title %>
-        </h3>
-        <p class="user-name">
-          <%= post.text %>
-        </p>
+  <!-- Posts Section -->
+  <% @posts.each do |post| %>
+    <div class="row">
+      <div class="col-12 post-item">
+        <div class="p-3 mt-2 border rounded">
+          <h6>Post <%= post.id %></h6>
+          <h3><%= post.title %></h3>
+          <p><%= post.text %></p>
+          <p class="text-muted text-right">Comments: x, Likes: y</p>
+        </div>
       </div>
```

Comment:
As shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png), we are expected to use the `#recent_post` method to render the user's 3 most recent post in the user show page 🤓. 

Your design
![image](https://github.com/sergio-usma/rails_blog_pj/assets/78251927/b347fb3e-348b-4cad-a8a0-837a2f75c8b5)

Wireframe
![image](https://github.com/sergio-usma/rails_blog_pj/assets/78251927/1d2377d9-b2d8-4135-bc75-67887b578225)

e.g 
```ruby
<% @user.recent_post.each do |post| %>
```
Please correct that 💪🛠

Reply:
Done!